docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+
+  mysql_server:
+    image: mysql:latest
+    restart: always
+    environment:
+      - MYSQL_DATABASE=process.env.MYSQL_DATABASE
+      - MYSQL_USER=process.env.MYSQL_USER
+      - MYSQL_PASSWORD=process.env.MYSQL_PASSWORD
+      - MYSQL_ROOT_PASSWORD=process.env.MYSQL_ROOT_PASSWORD
```

Comment:
tutaj process jest zla praktyka
https://stackoverflow.com/questions/42139605/how-do-you-manage-secret-values-with-docker-compose-v3-1
jest cos takiego jak secrety w composie, ale nie pierdol sie z tym narazie i wpisz z palca te deafultuwe

Reply:
w sumie da sie jeszcze tak 
https://www.youtube.com/watch?v=0JXyJOwVFfo&t=74s
https://www.youtube.com/watch?v=B2SrBzj4nTM
i na ten moment chyba najlepsze rozwiazanie